pkg/scafall.go

```diff
@@ -0,0 +1,69 @@
+// Scafall creates new source projects from project templates.  Project
+// templates are stored in git repositories and new source projects are created
+// on your local filesystem.
+package scafall
+
+// Scafall allows programmatic control over the default values for variables.
+// Any provided Arguments cause prompts for the same variable name to be skipped.
+type Scafall struct {
+	URL          string
```

Comment:
I know this isn't clarified in the RFC, but just thinking would URL be any valid `git` URL, i.e. can I pass in a local path as well for local dev of a template? Maybe I would use this/`pack` for tests of my template if I'm building a libcnb library.